### PR TITLE
Added spinner code page

### PR DESCRIPTION
**Pull request by raquelarrojo**
Closes [#1045](https://github.com/dxc-technology/halstack-react/issues/1045)

### CODE REVIEW

File: website/pages/components/spinner/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Head from "next/head";
 import type { ReactElement } from "react";
-import SpinnerUsagePage from "../../../screens/components/spinner/usage/SpinnerUsagePage";
+import SpinnerCodePage from "../../../screens/components/spinner/code/SpinnerCodePage";
 import SpinnerPageLayout from "../../../screens/components/spinner/SpinnerPageLayout";
 
 const Usage = () => {
```
Comment (Lisa159) [inline website/pages/components/spinner/index.tsx:6]: Change Usage to Index

File: website/screens/components/spinner/usage/SpinnerUsagePage.tsx [hunk 2]
```diff
@@ -5,6 +5,8 @@ import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
 import Figure from "@/common/Figure";
 import Code from "@/common/Code";
 import DocFooter from "@/common/DocFooter";
+import Example from "@/common/example/Example";
+import determinateIndeterminate from "./examples/determinateIndeterminate";
 import determinateImage from "./images/spinner_determinate_indeterminate.png";
```
Comment (Lisa159) [inline website/screens/components/spinner/usage/SpinnerUsagePage.tsx:10]: Remove unused import.

**Comment by Lisa159** [comment]
In SpinnerUsagePage in the line `The text of the action it is not mandatory but recommendable.`, I think it would be better to remove `it` from the middle of the sentence. It should be like: `The text of the action is not mandatory but recommendable.`.

**Comment by Lisa159** [comment]
In SpinnerUsagePage in the fourth point, it should be refactor the part where is said `use a medium or small spinner place the spinner in that part of the page.` to `use a medium or small spinner in that part of the page.`

File: website/pages/components/spinner/index.tsx (hunk 1 above)
Comment (raquelarrojo) [reply website/pages/components/spinner/index.tsx]: Done!

File: website/screens/components/spinner/code/SpinnerCodePage.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  {
+    title: "Props",
+    content: (
+      <DxcTable>
+        <tr>
+          <th>Name</th>
+          <th>Default</th>
+          <th>Description</th>
+        </tr>
+
```
[the remark this answers is not in the document]
Comment (raquelarrojo) [reply website/screens/components/spinner/code/SpinnerCodePage.tsx]: Done!

File: website/screens/components/spinner/usage/SpinnerUsagePage.tsx (hunk 2 above)
Comment (raquelarrojo) [reply website/screens/components/spinner/usage/SpinnerUsagePage.tsx]: Done!

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**